### PR TITLE
schema: update schematron rule for mixed beams

### LINKED ISSUE #690: Make Schematron rule for mixed beams a warning
The scheamtron rule `check_beam_place` requires all notes/chords within `<beam place="mixed">` to have a `@stem.dir`. 
This doesn't make sense, as you may want to advise the engraver to put a beam between the note heads without giving explicit stem directions. 
Also, it is not available with `beamSpan`, which isn't very consistent.

### PR DESCRIPTION
**Pull request by rettinghaus**
closes #690

### CODE REVIEW

File: source/modules/MEI.cmn.xml [hunk 1]
```diff
@@ -267,18 +267,15 @@
...
-              <sch:assert
-                test="count(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'][@stem.dir]) = count(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'])"
-                >Stem direction must be specified for all notes and chords under the
+              <sch:assert test="count(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'][@stem.dir]) = count(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'])" role="warning"
+                >Stem directions should be specified for all notes and chords under the
                 beam.</sch:assert>
-              <sch:assert
-                test="count(distinct-values(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord']/@stem.dir)) &gt; 1"
+              <sch:assert test="count(distinct-values(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord']/@stem.dir)) != 1"
```
Comment (musicEnfanthen) [inline source/modules/MEI.cmn.xml:273]: This would also be true for no stems, right?

File: source/modules/MEI.cmn.xml [hunk 2]
```diff
@@ -267,18 +267,15 @@
...
             <sch:rule
-              context="mei:beam[@place eq 'mixed' and (descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'][@staff != ./@staff] or descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'][@staff != ancestor::mei:staff/@n])                 and count(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'][@stem.dir]) = count(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'])]">
-              <sch:assert
-                test="count(distinct-values(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord']/@stem.dir)) &gt; 1"
+              context="mei:beam[@place eq 'mixed' and (descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'][@staff != ./@staff] or descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'][@staff != ancestor::mei:staff/@n]) and count(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord']/@stem.dir) = count(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord'])]">
+              <sch:assert test="count(distinct-values(descendant::mei:*[local-name() eq 'note' or local-name() eq 'chord']/@stem.dir)) != 1"
```
Comment (musicEnfanthen) [inline source/modules/MEI.cmn.xml:278]: Same here

File: source/modules/MEI.cmn.xml (hunk 1 above)
Comment (rettinghaus) [reply source/modules/MEI.cmn.xml:273]: It is now that having no `@stem.dir` at all throws a warning, but not any longer an error. It is still an error having the same `@stem.dir` encoded everywhere.

Comment (bwbohl) [reply source/modules/MEI.cmn.xml:273]: sounds as we decided in #690

File: source/modules/MEI.cmn.xml (hunk 2 above)
Comment (bwbohl) [reply source/modules/MEI.cmn.xml:278]: ibid.

**Review by bwbohl — APPROVED** [review]

**Review by musicEnfanthen — APPROVED** [review]

**Pull request merged**